rancher2/schema_node_template_openstack.go

```diff
@@ -56,11 +59,11 @@ func openstackConfigFields() map[string]*schema.Schema {
 		},
 		"region": {
 			Type:     schema.TypeString,
-			Required: true,
+			Optional: true,
```

Comment:
What's the reason to set `region` as optional argument??

Reply:
From Rancher and OpenStack point of view the region isn't required, not sure if it's because I'm using application credentials or just having only one region in my OpenStack setup. But after thinking twice it makes more sense to leave it as it is, to be on the safe side.